blocks/v2-feature-carousel/v2-feature-carousel.css

```diff
@@ -0,0 +1,76 @@
+:root {
+  --v2-feature-carousel-cards-gutter: 16px;
+  --v2-feature-carousel-card-width: 150px;
+  --v2-feature-carousel-card-height: 200px;
+  --v2-feature-carousel-icon-width: 48px;
+  --v2-feature-carousel-icon-height: 48px;
+  --v2-feature-carousel-svg-width: 24px;
+  --v2-feature-carousel-svg-height: 24px;
+}
+
+.v2-feature-carousel-container {
+  background-color: var(--c-grey-50);
+}
+
+.v2-feature-carousel-container .v2-feature-carousel-wrapper {
+  max-width: 1390px;
+  width: 100%;
+}
+
+.v2-feature-carousel__cards-list {
+  display: flex;
+  align-items: flex-start;
+  justify-content: center;
+  gap: var(--v2-feature-carousel-cards-gutter);
+  scroll-snap-type: x mandatory;
+  overflow: auto;
+  flex-flow: row nowrap;
+}
+
+.v2-feature-carousel__card {
+  display: flex;
+  flex: 0 0 auto;
+  width: var(--v2-feature-carousel-card-width);
+  height: var(--v2-feature-carousel-card-height);
+  padding: 24px;
+  flex-direction: column;
+  justify-content: space-between;
+  align-items: flex-start;
+  border-radius: 8px;
```

Comment:
is this border radius following the same structure as the rest of the components (image grid, masonrry grid, stories from the road....)? If so, use the variable `--border-radius`

Reply:
The border radius is same for smaller screen and its 10px for above 1200px. So accommodated changes accordingly.